### ISSUE TITLE
Issue with Autorank actulally ranking up

### ISSUE THREAD

**Issue by MrDrummer** [Armarr/Autorank-2#151]
Autorank is enabled, and no errors in the server log (as far as i can tell).
When a condition is met, (e.g. money=1000) it shows up as the condition has been met...
BUT it doesn't actually rank-up.

Bukkit version: craftbukkit-beta-3092
Advanced Config: http://pastebin.com/0iFnTEiC
Rank check interval: 1min (shortened for debug attempt)
Screenshot: http://img.ctrlv.in/img/14/11/30/547b961aadaa6.png

Basically, my player met the requirements, and it listed them as being met under /ar check, but it doesn't actually rank-up.

Sorry if i have not been detailed enough, any questions please ask :/

**Comment by Staartvin** [comment]
Closed due to inactivity.
